static/app/views/performance/landing/widgets/widgets/singleFieldAreaWidgetMetrics.tsx

```diff
@@ -71,6 +72,11 @@ export function SingleFieldAreaWidgetMetrics(
           environment={environment}
           query="transaction:foo" // TODO(metrics): make this dynamic once api is ready (widgetData.list.data[selectedListIndex].transaction)
```

Comment:
The API is ready to handle real queries now.

Reply:
oh nice...will add it

Reply:
We could add something like this until we visit https://getsentry.atlassian.net/browse/INGEST-679
```suggestion
          query={new MutableSearch(eventView.query).formatString()} // TODO(metrics): not all tags will be compatible with metrics
```